cpu/cpupower.py

```diff
@@ -31,9 +33,9 @@ class cpupower(Test):
     :avocado: tags=cpu,power,privileged
     """
 
+    @skipIf(not IS_POWER_NV, "This test is not supported on  PowerVM platform")
     def setUp(self):
-        if not os.path.exists('/sys/devices/system/cpu/cpu0/cpufreq'):
-            self.cancel('sysfs directory for cpufreq is unavailable.')
+
```

Comment:
Better to add some Docstring or least remove the empty line.

Reply:
sure